index.html

```diff
@@ -42,7 +83,7 @@ <h1 class="heading">Capital Jazz Fest 2023</h1>
             </div>
         </section>
     </header>
-    <section class="section2">
+    <section class="section2" id="program">
         <div class="title-container">
             <h2 class="title">Main Program</h2>
```

Comment:
- [ ] Kindly make sure to add a separator after the `Main Program` header so your design looks more like the provided design

|Your design|Provided design|
|-|-|
|![Screenshot_20230113_182853](https://user-images.githubusercontent.com/25227022/212381834-643eebcc-c741-4fb7-a019-658fa49f0132.png)|![Screenshot_20230113_182751](https://user-images.githubusercontent.com/25227022/212381845-6bef92d8-75c0-49a7-bcdf-8b257d45f273.png)|